src/main/java/com/redhat/pantheon/sling/PantheonRepositoryInitializer.java

```diff
@@ -49,9 +52,18 @@ private void initializeRepositoryACLs(SlingRepository slingRepository) throws Re
             } catch (AuthorizableExistsException aeex) {
                 log.info("Pantheon service account already exists");
             }
+
+            // JCR_WRITE and JCR_NODE_TYPE_MANAGEMENT are necessary to push content
+            // see: https://docs.adobe.com/docs/en/spec/jsr170/javadocs/jcr-2.0/javax/jcr/security/Privilege.html
             assignPermissionToPrincipal(s, "pantheon", "/content", "*", Privilege.JCR_ALL);
             assignPermissionToPrincipal(s,"pantheon-users","/content/repositories", null, Privilege.JCR_WRITE, Privilege.JCR_NODE_TYPE_MANAGEMENT);
             assignPermissionToPrincipal(s,"pantheon-users","/content/modules", null, Privilege.JCR_WRITE, Privilege.JCR_NODE_TYPE_MANAGEMENT);
+            // this is another way to do the above
+            AccessControlUtils.addAccessControlEntry(s,
+                    "/content/sandbox",
+                    AccessControlUtils.getPrincipal(s, "pantheon-users"),
+                    privilegesFromNames(s, Privilege.JCR_WRITE, Privilege.JCR_NODE_TYPE_MANAGEMENT),
+                    true);
```

Comment:
Not sure if this new way is working for us. I did some testing on this. demo user can't push to content/sandbox. It can still push to content/repositories. 

Here are my test notes:
// 500 error when pushing content as demo user. Demo user can create new modules in the web UI without issues. 
(py3) [llu@llu test_repo {upload_test *%}]$ pantheon push -b --server http://pantheon2.usersys.redhat.com:8080/ -u demo -p 
Using server: http://pantheon2.usersys.redhat.com:8080
Using sandbox: demo
--------------
/work/cp/test_repo/modules/docinfo.xml
500 Server Error
/work/cp/test_repo/modules/master.adoc
500 Server Error
58 additional files detected but not uploaded. Only files specified in pantheon2.yml are handled for upload.
Finished!


// When passing both '-b' and '--repository' options, '--repository' is ignored in the operation. The content is created is sandbox/admin in the backend. Do we need inform the end user about ignoring '--repository' during the push operation? 
(py3) [llu@llu test_repo {upload_test *%}]$ pantheon push -b --server http://pantheon2.usersys.redhat.com:8080/ -u admin -p --repository rhelrepo
Using server: http://pantheon2.usersys.redhat.com:8080
Using sandbox: admin
--------------
/work/cp/test_repo/modules/docinfo.xml
200 OK
/work/cp/test_repo/modules/master.adoc
200 OK
58 additional files detected but not uploaded. Only files specified in pantheon2.yml are handled for upload.
Finished!



// demo user can still push to content/repositories
(py3) [llu@llu test_repo {upload_test *%}]$ pantheon push  --server http://pantheon2.usersys.redhat.com:8080/ --repository rhelrepo
Using server: http://pantheon2.usersys.redhat.com:8080
Using repository: rhelrepo
--------------
/work/cp/test_repo/modules/docinfo.xml
201 Created
/work/cp/test_repo/modules/master.adoc
201 Created
58 additional files detected but not uploaded. Only files specified in pantheon2.yml are handled for upload.
Finished!